src/esp/sim/Simulator.h

```diff
@@ -27,6 +27,9 @@ class SemanticScene;
 namespace gfx {
 class Renderer;
 }  // namespace gfx
+namespace physics {
+enum class MotionType;
```

Comment:
```suggestion
enum class MotionType: int;
```

I wonder why compilers don't warn here? I'd always specify the type as well to be safe, otherwise if the actual `MotionType` definition has 64-bit values, this forward declaration would be incorrect.